### DISCUSSION TITLE
Enhancement Proposal: Better folder structure for starter app

### DISCUSSION THREAD

**Discussion by Izhaki** [vercel/turborepo#378] [Ideas]
### Describe the feature you'd like to request

Currently, when you run `npx create-turbo@latest` the folder structure is as follows:

```
apps
  docs
  web
packages
  config
  tsconfig
  ui
```

Thing is, both `config` and `tsconfig` are dev packages - they are never published, bundled, or consumed (non dev) with other (proper) packages.

### Describe the solution you'd like

Upon `npx create-turbo@latest`:

```
apps
  docs
  web
packages
  ui
config
  eslint
  typescript
```

### Describe alternatives you've considered

Doing it myself manually - totally doable.

**Comment by anthonyshew** [comment]
Mostly preferential! This happens to be my preference as well but the idea is we'd like to keep things as simple as sensible in the starters.
